common/static/common/js/spec_helpers/discussion_spec_helper.js

```diff
@@ -53,7 +53,7 @@
         DiscussionSpecHelper.makeCourseSettings = function() {
             return new DiscussionCourseSettings({
                 category_map: {
-                    children: ['Test Topic', 'Other Topic'],
+                    children: [['Test Topic', 'entry'], ['Other Topic', 'entry']],
```

Comment:
You are converting this to List of Lists here and in `lms/djangoapps/django_comment_client/tests/test_utils.py:399` you are making list of tuples. Please use same verywhere.